connect-examples/v1/php/payments-report.php

```diff
@@ -49,29 +49,30 @@ function getLocationIds() {
 }
 
 # Retrieves all of a merchant's payments from 2015
-function get2015Payments($location_ids) {
+function getPayments($location_ids) {
```

Comment:
nice :)

Reply:
Made me notice there was still "2015" in the comments. Removed those :P thanks.